### ISSUE TITLE
Cannot download v0.0.12 from npm

### ISSUE THREAD

**Issue by jshthornton** [tonyhb/redux-ui#29]
```
npm ERR! fetch failed https://registry.npmjs.org/redux-ui/-/redux-ui-0.0.12.tgz
npm WARN retry will retry, error on last attempt: Error: fetch failed with status code 404
```

v0.0.11 still works fine.

**Comment by jakubrohleder** [comment]
Same here! Quick workaround is to use github address `npm install https://github.com/tonyhb/redux-ui --save`

**Comment by tonyhb** [comment]
Awesome, thanks for the heads up guys. Going to take a look at this and fix
it now.

**Comment by tonyhb** [comment]
Something must have gone wrong with publishing v0.0.12; it was in my npm cache locally which is why it went unnoticed. Fixed, had to publish v0.0.13. Thanks!

**Comment by tonyhb** [comment]
@jshthornton - NPM fucked up, just FYI: http://status.npmjs.org/incidents/7n06gj3f83tz
